addons/mail/static/src/core/common/chat_bubble.xml

```diff
@@ -29,10 +29,14 @@
                     <t t-set="message" t-value="message"/>
                 </t>
                 <t t-if="message.hasOnlyAttachments">
+                    <MessageSeenIndicator t-if="message.showSeenIndicator(thread)" message="message" thread="thread" />
                     <i class="fa me-1" t-att-class="message.previewIcon"/>
                     <t t-out="message.previewText" />
                 </t>
-                <t t-else="" t-out="previewText" />
+                <t t-else="">
+                    <MessageSeenIndicator t-if="message.showSeenIndicator(thread)" message="message" thread="thread" />
+                    <t t-out="message.previewText"/>
+                </t>
             </div>
```

Comment:
`<MessageSeendIndicator/>` is copied twice.
Could be moved before the `t-if`/`t-else` to have it once.